tests/compilation_tests/robot_test/traverse_standard_test.py

```diff
@@ -15,11 +15,46 @@
 gps_noise_range = .3
 
 
+class TestMoveToTargetNode(unittest.TestCase):
```

Comment:
wanted to check if these are the existing test cases that we were failing originally?

Reply:
I don't think we failed any, just had unexpected behavior